.gitignore

```diff
@@ -160,3 +160,9 @@ cython_debug/
 #.idea/
 
 /src/qw/_version.py
+
+# poetry
+poetry.lock
```

Comment:
oh could remove this

Reply:
Ah, should I add poetry.lock to the repo?
I suppose that's sensible.

Reply:
IIRC their suggestion is if you're writing an application, include the lock. If its a library then maybe don't